python-api-examples/offline-decode-files.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python3
+
+"""
+This file demonstrates how to use sherpa-onnx Python API to transcribe
+file(s) with a streaming model.
+
+paraformer Usage:
+    ./python-api-examples/offline-decode-files.py  \
+      --tokens=/path/to/tokens.txt \
+      --paraformer=/path/to/paraformer.onnx \
+      --num-threads=2 \
+      --decoding-method=greedy_search \
+      --debug=false \
+      --sample-rate=16000 \
+      --feature-dim=80 \
+      /path/to/0.wav \
+      /path/to/1.wav
+
+transducer Usage:
+    ./python-api-examples/offline-decode-files.py  \
+      --tokens=/path/to/tokens.txt \
+      --encoder=/path/to/encoder.onnx \
+      --decoder=/path/to/decoder.onnx \
+      --joiner=/path/to/joiner.onnx \
+      --num-threads=2 \
+      --decoding-method=greedy_search \
+      --debug=false \
+      --sample-rate=16000 \
+      --feature-dim=80 \
+      /path/to/0.wav \
+      /path/to/1.wav
+
+Please refer to
+https://k2-fsa.github.io/sherpa/onnx/index.html
+to install sherpa-onnx and to download the pre-trained models
+used in this file.
+"""
+import argparse
+import time
+import wave
+from pathlib import Path
+from typing import Tuple
+
+import numpy as np
+
+from _sherpa_onnx import (
```

Comment:
Could you use `sherpa_onnx.OfflineRecognizer` to replace
the classes you are using from `_sherpa_onnx`?

`_sherpa_onnx` should be invisible to users.